blueocean-git-pipeline/src/test/java/io/jenkins/blueocean/blueocean_git_pipeline/GitPipelineCreateRequestTest.java

```diff
@@ -41,27 +43,29 @@ public void createSomeStuff() throws Exception {
 
     @Test
     public void createPipeline() throws UnirestException, IOException {
-        User user = login("vivek", "Vivek Pandey", "vivek.pandey@gmail.com");
-        Map r = new PipelineBaseTest.RequestBuilder(baseUrl)
-            .status(201)
-            .jwtToken(getJwtToken(j.jenkins, user.getId(), user.getId()))
-            .crumb( crumb )
-            .post("/organizations/jenkins/pipelines/")
-            .data(ImmutableMap.of("name", "pipeline1",
-                "$class", "io.jenkins.blueocean.blueocean_git_pipeline.GitPipelineCreateRequest",
-                "scmConfig", ImmutableMap.of("id", GitScm.ID, "uri", sampleRepo.toString())))
-            .build(Map.class);
-        assertNotNull(r);
-        assertEquals("pipeline1", r.get("name"));
+        User user = user("vivek", "Vivek Pandey", "vivek.pandey@gmail.com");
+        try (ACLContext ctx = ACL.as(user)) {
+            Map r = new PipelineBaseTest.RequestBuilder(baseUrl)
+                    .status(201)
+                    .jwtToken(getJwtToken(j.jenkins, user.getId(), user.getId()))
```

Comment:
Most of these changes are whitespace, converting to a block so as to reset the authentication at the end. (Setting authentication and never resetting it is very poor form, even in a test.) Turn on the WS-ignoring diff mode to review better.

Note that I have no idea how many of these tests actually relied on `Jenkins.authentication` being switched to the test user; the utility methods did this across the board, but I suspect many just relied on such a user _existing_. Or they may not have relied on the user at all; many of these tests (unlike this one) have no apparent relationship to authentication or access control.